src/components/PackageDocs/index.tsx

```diff
@@ -42,6 +43,10 @@ export function appendItem(
   }
 }
 
+// We want the nav to be sticky, but it should account for the sticky heading as well
+// The calculation here is heading height (72px) + margin-top (16px).
+const TOP_OFFSET = "88px";
```

Comment:
I think this should either be a constant that is used in both components to set the header height and the navbar offset, or they should both just use common theme values ie `height={4}` & `top={4}`.

I'm not sure if we even want the header to be globally sticky have we really defined that?

Reply:
I believe it is a better experience to keep a sticky header. Could def go about setting a theme value for the header height so that this interaction uses the theme